### DISCUSSION TITLE
Tour pauses

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1025] [General]
Hi
How can I make tour pauses not visible on map?

**Comment by wolfgang-ch** [comment]
Tour pauses cannot currently be hidden in a map

**Reply by wolfgang-ch** [reply]
Example: The pause between the left slider `L` and right slider `R` is painted in the map

![image](https://user-images.githubusercontent.com/1283445/224958988-a9444efb-6766-4361-96fd-c2cd6d2b126d.png)

**Comment by wolfgang-ch** [comment]
When tours are painted with dot's, then the pauses are visible

![image](https://user-images.githubusercontent.com/1283445/234483792-fd6b47d5-aeae-4b1e-9f44-b37e117b1440.png)
